xchange-stream-ftx/src/main/java/info/bitrich/xchangestream/ftx/FtxStreamingAdapters.java

```diff
@@ -29,11 +29,17 @@
 import org.knowm.xchange.ftx.dto.marketdata.FtxTradeDto;
 import org.knowm.xchange.ftx.dto.trade.FtxOrderFlags;
 import org.knowm.xchange.instrument.Instrument;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class FtxStreamingAdapters {
 
+  private static final Logger LOG = LoggerFactory.getLogger(FtxStreamingAdapters.class);
+
   private static final ObjectMapper mapper = StreamingObjectMapperHelper.getObjectMapper();
-  /** Incoming values always has 1 trailing 0 after the decimal, and start with 1 zero */
+  /**
+   * Incoming values always has 1 trailing 0 after the decimal, and start with 1 zero
+   */
   private static final ThreadLocal<DecimalFormat> dfp =
       ThreadLocal.withInitial(() -> new DecimalFormat("0.0#######"));
```

Comment:
this is the place where this issue should be fixed 

https://docs.oracle.com/javase/7/docs/api/java/text/DecimalFormat.html

we should force the proper separator here instead of fixing in with string replace